tests/test_arrow_dataset.py

```diff
@@ -1341,6 +1341,18 @@ def func(example):
                     )
                     self.assertListEqual(dset_test[0]["tensor"], [1, 2, 3])
 
+    def test_map_input_columns(self, in_memory):
```

Comment:
I think this test is already passing if `output_all_columns=False`

You may need to add another column than "filename" to have it properly tested